js/src/product-feed/review-request/review-request-notice.js

```diff
@@ -66,17 +66,19 @@ const ReviewRequestNotice = ( {
 				</Flex>
 			</FlexItem>
 			<FlexItem className="gla-review-request-notice__button">
-				{ accountReviewStatus.requestButton && (
-					<AppButton
-						isPrimary
-						onClick={ onRequestReviewClick }
-						disabled={ !! account.cooldown }
-						text={ __(
-							'Request review',
-							'google-listings-and-ads'
-						) }
-					/>
-				) }
+				{ accountReviewStatus.requestButton &&
+					( account.cooldown ||
+						account.reviewEligibleRegions?.length > 0 ) && (
```

Comment:
💅 
Not sure if there is any case that the `account.reviewEligibleRegions` will not be an array? Otherwise, the optional chaining operator should be unnecessary.